README.md

```diff
@@ -1334,6 +1334,66 @@ Reset the location service on real device since Appium 1.22.0.
 It could delay a few seconds to reflect the location by the system.
 It raises an error if the device is simulator or an error occurred during the reset.
 
+### mobile: enableConditionInducer
+
+Important: Device conditions are available for real devices running iOS 13.0 and later.
+
+You can create a device condition on a connected device to test your app under adverse conditions, such as poor network connectivity or thermal constraints.
```

Comment:
You can create a device condition on a connected device -> You can create a condition on a connected device